buster/formatter/__init__.py

```diff
@@ -1,6 +1,13 @@
-from .base import Formatter
-from .html import HTMLFormatter
-from .markdown import MarkdownFormatter
-from .slack import SlackFormatter
+from .base import ResponseFormatter
```

Comment:
To remove the import from base above:
```suggestion
from .base import Response, ResponseFormatter, Source
```